xGame/Java/xgame/src/main/java/com/xgame/restservice/controller/ChessMatchController.java

```diff
@@ -4,20 +4,22 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PatchMapping;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.xgame.common.enums.MatchOutcome;
 import com.xgame.common.viewmodels.MatchViewModel;
 import com.xgame.service.interfaces.IChessMatchService;
 
+@CrossOrigin
 @RestController
 public class ChessMatchController {
 
 	@Autowired
 	IChessMatchService matchService;
 	
-	@PostMapping("/match")
+	@PostMapping("/match/create")
```

Comment:
Is there a particular reason you added "/create" here? Not a big deal, but creation is usually implied by any methods that use POST

Reply:
No, I just misunderstood something